src/test/java/com/karankumar/bookproject/ui/goal/ReadingGoalViewTest.java

```diff
@@ -118,22 +120,22 @@ void testSetGoalButtonTextUpdatesWhenGoalUpdates() {
     }
 
     @Test
-    void testTargetMetMessageNotShownWhenGoalNotMet() {
+    void notShownTargetMetMessageWhenGoalNotMet() {
```

Comment:
Typo here: this should be `notShow`, not `notShown` (it made sense before but not here)